Add kubectl-accurate template list sub command. (#9)
cybozu-go/accurate: docs/kubectl-accurate.md

To unset the type, specify `none` as the type.

### `template list [TEMPLATE]`

List template namespace trees hierarchically.

> If TEMPLATE is not given, all root namespaces and their children will be shown.

Reviewer: ```suggestion
If TEMPLATE is not given, all template namespaces are shown hierarchically.
```

Author: Sorry, Thank you comments.
